css/icons.css

```diff
@@ -71,10 +71,12 @@
  * .app-Talk rules above.
  * "forced-white" needs to be included in the class name as the Avatar does
  * not accept several classes. */
+.mention-bubble__icon.icon-group-forced-white,
 .tribute-container .icon-group-forced-white {
 	background-image: url(../img/icon-contacts-white.svg);
 }
 
+.mention-bubble__icon.icon-user-forced-white,
 .tribute-container .icon-user-forced-white {
 	background-image: url(../img/icon-user-white.svg)
 }
```

Comment:
**thought:** _Not a part of this PR_

`mention-bubble__icon` is some internal class from `@nextcloud/vue` lib and not a part of public API. As this file with libraries overrides became bigger and bigger, I'd think about more safe alternative.